src/site/content/en/blog/aspect-ratio/index.md

```diff
@@ -0,0 +1,211 @@
+without cropping.
+
+## Maintaining aspect ratio with `padding-top`
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="aspectratio-current.jpg" alt="">
+  <figcaption class="w-figcaption">
+    Using `padding-top` to set a 1:1 aspect ratio on post preview images within a carousel.
+  </figcaption>
+</figure>
+
+In order to make these more responsive, we can use aspect-ratio. This allows for
+us to set a specific ratio size and base the rest of the media on an individual
+axis (height or width).
+
+A currently well-accepted cross-browser solution for maintaining aspect ratio
+based on an image's width is known as the “Padding-Top Hack". This solution
+requires a parent container and an absolutely placed child container. One would
+then calculate the aspect ratio as a percentage to set as the `padding-top`. For
+example:
+
+1:1 aspect ratio = 1 / 1 = 1 = `padding-top: 100%` 4:3 aspect ratio = 3 / 4 =
+0.75 = `padding-top: 75%` 3:2 aspect ratio = 2 / 3 = 0.66666 = `padding-top:
+66.67%` 16:9 aspect ratio = 9 / 16 = 0.5625 = `padding-top: 56.25%`
+
+Now that we have identified the aspect ratio value, we can apply that to our
+parent container. Consider the following example:
+
+```html
+<div class="container">
+  <img class="media" src="..." alt="">
+</div>
+```
+
+We could then write the following CSS:
+
+```css
+.container {
+  position: relative;
+  width: 100%;
+  padding-top: 56.25%; /* 16:9 Aspect Ratio */
+}
+
+.media {
+  position: absolute;
+  top: 0;
+}
+```
+
+## Maintaining aspect ratio with `aspect-ratio`
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="aspectratio-future.jpg" alt="">
+  <figcaption class="w-figcaption">
+    Using `aspect-ratio` to set a 1:1 aspect ratio on post preview images within a carousel.
+  </figcaption>
+</figure>
+
+Unfortunately, calculating these `padding-top` values is not very intuitive, and
+requires some additional overhead and positioning. With the new intrinsic
+`aspect-ratio` [CSS property](https://www.w3.org/TR/css-sizing-4/#aspect-ratio),
+coming to Chromium 88, Safari Technical Preview 118, and Firefox __, the
+language for maintaining aspect ratios is much more clear.
+
+With the same markup, we can replace: `padding-top: 56.25%` with `aspect-ratio:
+16 / 9`, setting `aspect-ratio` to a specified ratio of `width` / `height`.
+
+<div class="w-columns">
+{% Compare 'worse', 'Using padding-top' %}
+```css
+.container {
+  width: 100%;
+  padding-top: 56.25%;
+}
+```
+{% endCompare %}
+
+{% Compare 'better', 'Using aspect-ratio' %}
+```css
+.container {
+  width: 100%;
+  aspect-ratio: 16 / 9;
+}
+```
+{% endCompare %}
+</div>
+
+Using `aspect-ratio` instead of `padding-top` is much more clear, and does not
+overhaul the padding property to do something outside of its usual scope.
+
+Now, not only is the aspect ratio much more clear, but this new property also
+adds the ability to set aspect ratio to `auto`, where “replaced elements with an
```

Comment:
```suggestion
This new property also adds the ability to set aspect ratio to `auto`, where “replaced elements with an
```
I don't think you need to reiterate that because you just said it in the last paragraph